application/modules/invoices/models/Mdl_invoices.php

```diff
@@ -64,7 +84,7 @@ public function default_select()
             IFnull(ip_invoice_amounts.invoice_paid, '0.00') AS invoice_paid,
             IFnull(ip_invoice_amounts.invoice_balance, '0.00') AS invoice_balance,
             ip_invoice_amounts.invoice_sign AS invoice_sign,
-            (CASE WHEN ip_invoices.invoice_status_id NOT IN (1,4) AND DATEDIFF(NOW(), invoice_date_due) > 0 THEN 1 ELSE 0 END) is_overdue,
+            (CASE WHEN ip_invoices.invoice_status_id NOT IN (1,5) AND DATEDIFF(NOW(), invoice_date_due) > 0 THEN 1 ELSE 0 END) is_overdue,
```

Comment:
Let's re-think this one. Should it be 1, 4, 5?

Reply:
Oh yes you are right,